src/docs/install.md

```diff
@@ -17,47 +17,46 @@ After you have pants installed, you'll need to
 Recommended Installation
 ------------------------
 
-To set up pants in your repo, we recommend installing our self-contained `pants` bash script
-in the root (ie, "buildroot") of your repo:
+To set up Pants in your repo, we recommend installing our self-contained `pants` bash script
+in the root (i.e. the "buildroot") of your repo:
 
     :::bash
-    curl -L -O https://pantsbuild.github.io/setup/pants && chmod +x pants && touch pants.ini
+    curl -L -O https://pantsbuild.github.io/setup/pants && chmod +x pants
 
-The first time you run the new `./pants` script it will install the latest version of pants (using
-virtualenv) and then run it.  It's recommended though, that you pin the version of pants.  To do
-this, first find out the version of pants you just installed:
+The first time you run the new `./pants` script, it will install the latest version of Pants (using
```

Comment:
Is it worth merging this sentence onto the end of the `Start by running...` sentence below? It's a bit of an implementation detail.